lib/utils/string_extension.dart

```diff
@@ -89,4 +89,34 @@ extension StringCasingExtension on String {
     final String? firstValidLink = matches.firstWhere((link) => AnyLinkPreview.isValidLink(link!));
     return firstValidLink;
   }
+
+  // Removes markdowned links from a string based on the unformatted text
+  // Workaround for content['formatted_body'] which formats urls in a way that makes them unusable
```

Comment:
Do you make a request change to upstream library

Reply:
no, didn't really look into it